src/lib.rs

```diff
@@ -45,9 +45,9 @@ where
 {
     // For example, there is 11 hashes [A..K] are used to generate a merkle tree:
     //
-    //      A_B C_D E_F
+    //      F_G H_I J_K
```

Comment:
I think the top should be root, after you change the logic.

Reply:
https://github.com/cryptape/static-merkel-tree/pull/11